Formula/cloudflare-wrangler2.rb

```diff
@@ -0,0 +1,24 @@
+require "language/node"
+
+class CloudflareWrangler2 < Formula
+  desc "CLI tool for Cloudflare Workers"
+  homepage "https://github.com/cloudflare/wrangler2"
+  url "https://registry.npmjs.org/wrangler/-/wrangler-2.6.2.tgz"
+  sha256 "f2a5a803db8c5e8eaa2d75affc12330deb8cef33ec01c29bbc0ebdb3bb3b4985"
+  license any_of: ["Apache-2.0", "MIT"]
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/wrangler2"]
+
+    # Replace universal binaries with their native slices
+    deuniversalize_machos libexec/"lib/node_modules/wrangler/node_modules/fsevents/fsevents.node"
+  end
+
+  test do
+    system "#{bin}/wrangler2", "init", "--yes"
+    assert !(testpath/"wrangler.toml").zero?
```

Comment:
```suggestion
    assert_exists testpath/"wrangler.toml"
```

Reply:
My intention with using `.zero?` was to make sure it didn't just create an empty file, but if that is acceptable as a test, then I would be happy to go for your suggestion instead.

Reply:
I don't think there's too much difference between our options. So out of those two I think mine looks nicer. Maybe we can validate the toml with something like a `wrangler2 status`? That way we check it's actually valid instead of just existing.

Reply:
I found that we can do a dry-run of publishing a site. It will fail if the `wrangler.toml` file doesn't exist or is empty.